doc/dev/GETTING_STARTED.md

```diff
@@ -173,6 +192,15 @@ You can then simply run F3D from the command line:
 ./bin/f3d.app/Contents/MacOS/f3d
 ```
 
+### Running the test suite
+
+```
+cd build
+cmake -DBUILD_TESTING=ON ../src
```

Comment:
I am not sure about the instructions for Windows

Reply:
this is macOS, and it looks fine

Reply:
Thanks